rebound-android-playground/src/main/res/layout/activity_playground.xml

```diff
@@ -1,37 +1,30 @@
 <FrameLayout
   xmlns:android="http://schemas.android.com/apk/res/android"
-  xmlns:tools="http://schemas.android.com/tools"
-  tools:context=".Playground"
   android:id="@+id/root"
   android:layout_width="match_parent"
   android:layout_height="match_parent"
-  android:background="@android:color/black"
-  >
+  android:background="@android:color/black">
+
   <LinearLayout
     android:id="@+id/root_container"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:background="#ccc"
-    android:orientation="vertical"
-    >
-    <FrameLayout
+    android:orientation="vertical">
+
+    <ImageView
       android:layout_width="match_parent"
       android:layout_height="wrap_content"
+      android:src="@drawable/rebound_icon"
       android:background="#1194e7"
-      android:padding="40dp"
-      >
-      <ImageView
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:src="@drawable/rebound_icon"
-        android:layout_gravity="center"
-        />
-    </FrameLayout>
+      android:layout_gravity="center"
+      android:padding="40dp"/>
+
     <ListView
       android:id="@+id/list_view"
       android:layout_width="match_parent"
       android:layout_height="match_parent"
-      android:layout_weight="1"
-      />
+      android:background="#ccc"/>
```

Comment:
you may want to consider extracting this color value to the colors.xml file (to be consistent with the rest of the file, for example line 6)